tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -525,9 +523,9 @@ public function testEnsureTableOptionsAreReflectedInMetadata(): void
 CREATE TABLE test_table_metadata(
   col1 INT NOT NULL AUTO_INCREMENT PRIMARY KEY
 )
-COLLATE utf8_general_ci
+COLLATE utf8mb4_general_ci
 ENGINE InnoDB
-ROW_FORMAT COMPRESSED
```

Comment:
This causes the following error on MariaDB 10.7:

> InnoDB refuses to write tables with ROW_FORMAT=COMPRESSED or KEY_BLOCK_SIZE.

When testing schema introspection, we don't care about the specific value of the row format, so we can use any other valid value.